### ISSUE TITLE
Buffer not working

### ISSUE THREAD

**Issue by damianobarbati** [rt2zz/redux-persist#270]
@rt2zz reporting the bug here instead of redux-action-buffer as you said :)
I get  the following by redux persist when the only action fired is the classic redux @@INIT.
```
 redux-persist/autoRehydrate: 1 actions were fired before rehydration completed. This can be a symptom of a race
      condition where the rehydrate action may overwrite the previously affected state. Consider running these actions
      after rehydration:
    
autoRehydrate.js:65 redux-persist/autoRehydrate: sub state for key `auth` modified, skipping autoRehydrate.
```

```
import { createStore, combineReducers, applyMiddleware } from 'redux';
import { composeWithDevTools } from 'redux-devtools-extension/developmentOnly';
import { routerMiddleware, routerReducer } from 'react-router-redux';
import thunkMiddleware from 'redux-thunk';
import promiseMiddleware from 'redux-promise';
import createLogger from 'redux-logger';
import { browserHistory } from 'react-router';

import { REHYDRATE } from 'redux-persist/constants'
import { persistStore, autoRehydrate } from 'redux-persist';
import createActionBuffer from 'redux-action-buffer';
import crosstabSync from 'redux-persist-crosstab';

import authReducer from './auth.reducer.js';

const reducers = {
	routing: routerReducer,
	auth: authReducer,
};
const finalReducer = combineReducers(reducers);

const loggerMiddleware = createLogger();
const finalMiddleware = applyMiddleware(
	routerMiddleware(browserHistory),
	thunkMiddleware,
	promiseMiddleware,
	createActionBuffer(REHYDRATE),
	loggerMiddleware,
);

const store = createStore(finalReducer, {}, composeWithDevTools(
	finalMiddleware,
	autoRehydrate({ log: true }),
));

export default store;
```

**Comment by rt2zz** [comment]
yes I will update to not warn on the init action

**Comment by rt2zz** [comment]
resolved in 4.2.0 ty!
